tests/Gaufrette/Functional/adapters/GoogleCloudClientStorage.php.dist

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
We're avoiding this way of running test. Could you update following the new recommended way (see #465)?